sal_interfaces/MTM2/MTM2_Commands.xml

```diff
@@ -101,4 +101,46 @@
     </item>
 </SALCommand>
 
+<SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_switchForceBalanceSystem</EFDB_Topic>
+    <Alias>switchForceBalanceSystem</Alias>
+    <Description>Switch the force balance (FB) system.</Description>
+    <item>
+        <EFDB_Name>status</EFDB_Name>
+        <Description>0: turn off FB. 1: turn on FB.</Description>
+        <IDL_Type>int</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALCommand>
+
+<SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_useZenithAngleFromMount</EFDB_Topic>
```

Comment:
So, the mount publishes Elevation and not Zenit Angle. But maybe it would be a better idea to make this more generic. Also, how would we switch back to using the on-board instruments? Based on  the payload this is just a "on" command. I would suggest we use a command like:

`selectInclinationTelemetrySource` and the payload would be `source` which can be an enumeration with the options like `ONBOARD` and `MOUNT`.

Reply:
Done in commit: 9c5cc3f. Thanks!